src/dotnet-dev-certs/Program.cs

```diff
@@ -73,8 +79,14 @@ public static int Main(string[] args)
                         var reporter = new ConsoleReporter(PhysicalConsole.Singleton, verbose.HasValue(), quiet.HasValue());
                         if (check.HasValue())
                         {
-                            return CheckHttpsCertificate(check, trust, reporter);
+                            return CheckHttpsCertificate(trust, reporter);
+                        }
+
+                        if (clean.HasValue())
```

Comment:
looks like if both clean and check flags will be specified, the tool will always do the "check" only. Should we rather show some error?

Reply:
I've added error checking above to weed out these cases